_hackathons/sydney.html

```diff
@@ -0,0 +1,17 @@
+---
+layout: hackathon
+sort-date: 2018-03-01
+_slug: hackathons-sydney
+title: Hackathons
+location: Sydney, Australia
+date_txt: March 9-11
+address: Sydney Startup Hub; Level 4/11 York St, Sydney, NSW 2000
+---
+
+<p></p><p>Co founder of holochain Art Brock is going to be in Sydney where on Thursday 3/7 he will speak on "Think Outside the Blocks" to explain how holochain overcomes the limitations of blockchain.  <a href="https://www.eventbrite.com/e/think-outside-the-blocks-tickets-43644250127"> Tickets are free</a>.
```

Comment:
Co-founder of Holochain, Art Brock, is going to be in Sydney where on Thursday, March 7th he will speak on "Think Outside the Blocks" to explain how Holochain overcomes the limitations of blockchain. 

Then, March 9th–11th, we'll have a hackathon to provide a guided hands-on experience of how easy it is to build powerful solutions with Holochain. We'll have working prototypes by the end of the weekend.

[Take out double spaces throughout]